src/components/Toast/ErrorToast.tsx

```diff
@@ -0,0 +1,39 @@
+import Link from "next/link"
+import { FC } from "react"
+import { toast } from "react-hot-toast"
+
+import { Toast, TransactionToastComponentProps } from "@/components/Toast"
+
+import { FortIconWarning } from "@/icons"
+
+export const ErrorToast: FC<TransactionToastComponentProps> = ({
+  message,
+  txHash,
+  ...toastProps
+}) => {
+  return (
+    <Toast
+      className="flex items-center bg-red-500 text-white"
+      onDismiss={() => toast.dismiss(txHash)}
```

Comment:
After thinking about this some more, I don't think I like the `txHash` as the ID because not every toast will receive a `txHash`.

Just use the toast ID from react-hot-toast.

If you're testing in `useEffect` (on mount) you will see multiple toasts because `useEffect` fires multiple times in React 18. In real-world scenarios, there won't be duplicates.